modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsPerformanceByAssignee.testcase

```diff
@@ -32,6 +32,7 @@ definition {
 	@description = "LPS-97666 - Verify that the names inside the steps filter component from Performance by Assignee card are breaking into another line when the component width exceeds 240px"
 	@priority = 3
 	test AssigneeNamesBreaksIntoAnotherLine {
+		property portal.acceptance = "defaulttest";
```

Comment:
@Luizcarlosqueiroz Looks good! Go ahead and forward a PR without this property. Thank you!

Reply:
Oh, I forgot to remove it. Thanks!